nitro-protocol/contracts/MultiAssetHolder.sol

```diff
@@ -56,11 +59,7 @@ contract MultiAssetHolder is IMultiAssetHolder, StatusManager {
         if (asset == address(0)) {
             require(msg.value == amount, 'Incorrect msg.value for deposit');
         } else {
-            // require successful deposit before updating holdings (protect against reentrancy)
-            require(
-                IERC20(asset).transferFrom(msg.sender, address(this), amountDeposited),
-                'Could not deposit ERC20s'
-            );
+            IERC20(asset).safeTransferFrom(msg.sender, address(this), amountDeposited);
```

Comment:
Do we know what happens if this call is made for USDT but the sender has less than `amountDeposited` tokens? Silent failure? Or a an error?

Reply:
Good question. Here's the verified source for USDT on mainnet https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7#code . 

On L185 there, we have a SafeMath style subtraction which will revert the transaction if the sender has insufficient tokens.